spec/models/rdv/send_webhook_job_spec.rb

```diff
@@ -0,0 +1,38 @@
+describe Rdv, type: :model do
```

Comment:
I think this belongs in the `rdv_spec.rb` file unless you extract all webhook behaviour into a separate module, which could be a good idea, I don't really have a strict opinion on that.